doc/tutorial/tutorial.mdk

```diff
@@ -1308,6 +1357,14 @@ By adding a type annotation we can give a _different_ type to `map`:
 This type is neither a subtype nor a supertype of the type F\*
 inferred on its own.
 
+~KK
+At this point I could not really figure out how the type inferred by F\* and the
+explicitly written type differ functionally. I suspect this is due to my limited
+knowledge of functional programming and has something to do with currying.
+Anyway, a small remark about this might make sense here.
+Also: why can or does F\* not infer the different type? 
+~
+

```

Comment:
The pre-condition is stronger, while the post-condition is weaker. 

So while the inferred type could take a Tot function as input (thus potentially being a sub-type), it does not output a Tot (list 'b). 
I believe it has to do with covariant and contravariant: https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science)#Function_types